starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +56,19 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  /**
+   * Test Create method.
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setVersion(VERSION);
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    this.inventoryDAO.create(inventory);
+
+    Query findByName = new Query(where("name").is(NAME));
+    Assert.assertEquals(inventory, this.mongoTemplate.findOne(findByName, Inventory.class));
```

Comment:
What other assertions can we use to compare inventory with the output of the create() method to make sure that it actually created the one we wanted it to?